src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -642,61 +644,32 @@ const DetailsFormLeft: React.FC<DetailsFormLeftProps> = ({
   const renderShowScheduleToggle = () => (
     <div className="toggle-room">
       <h4 className="italic input-header">Show Schedule</h4>
-      <label id={"showSchedule"} className="switch">
-        <input
-          type="checkbox"
-          id={"showSchedule"}
-          name={"showSchedule"}
-          ref={register}
-          defaultChecked={DEFAULT_SHOW_SCHEDULE}
-        />
-        <span className="slider round"></span>
-      </label>
+      <Toggler
+        name="showSchedule"
+        forwardedRef={register}
+        defaultChecked={DEFAULT_SHOW_SCHEDULE}
+      />
     </div>
   );
 
   const renderShowGridToggle = () => (
     <div className="toggle-room">
       <h4 className="italic input-header">Show grid layout</h4>
-      <label id={"showGrid"} className="switch">
-        <input
-          type="checkbox"
-          id={"showGrid"}
-          name={"showGrid"}
-          ref={register}
-        />
-        <span className="slider round"></span>
-      </label>
+      <Toggler name="showGrid" forwardedRef={register} />
     </div>
   );
 
   const renderShowBadgesToggle = () => (
     <div className="toggle-room">
       <h4 className="italic input-header">Show badges</h4>
-      <label id={"showBadges"} className="switch">
-        <input
-          type="checkbox"
-          id={"showBadges"}
-          name={"showBadges"}
-          ref={register}
-        />
-        <span className="slider round"></span>
-      </label>
+      <Toggler name="showBadges" forwardedRef={register} />
     </div>
   );
 
   const renderShowZendeskToggle = () => (
```

Comment:
```suggestion
  // @debt pass the header into Toggler's 'label' prop instead of being external like this
  const renderShowZendeskToggle = () => (
```